docs/src/capi/tutorial/getting-started.md

```diff
@@ -94,7 +94,49 @@ export AWS_B64ENCODED_CREDENTIALS=$(clusterawsadm bootstrap credentials encode-a
 ```
 
 You are now all set to deploy the AWS CAPI infrastructure provider.
+````
+
+````{group-tab} MAAS
+Start by setting up environment variables defining the MAAS credentials, if
+these are not already defined:
+
+```
+export MAAS_API_KEY="<maas-api-key>"
+export MAAS_ENDPOINT="http://<maas-endpoint>/MAAS"
+export MAAS_DNS_DOMAIN="<maas-dns-domain>"
```

Comment:
Is there anything more necessary to do if maas_endpoint is `https`?  Is that supported?

Reply:
AFAIK upstream provider does not suggest/do anything special for `https` endpoints. The provider uses a simple go client which performs http requests through the standard library so I expect things to work properly. Seems like the client uses `InsecureSkipVerify: true` which can be seen [here](https://github.com/spectrocloud/maas-client-go/blob/072142d8aa50f39a4f11cd59e3a83a53f7f047c4/maasclient/client.go#L244).